packages/js/product-editor/src/components/attribute-input-field/attribute-input-field.tsx

```diff
@@ -107,12 +108,7 @@ export const AttributeInputField: React.FC< AttributeInputFieldProps > = ( {
 					.startsWith( inputValue.toLowerCase() )
 		);
 
-		if (
-			inputValue.length > 0 &&
-			! allItems.find(
-				( item ) => item.name.toLowerCase() === inputValue.toLowerCase()
-			)
-		) {
+		if ( inputValue.length > 0 ) {
```

Comment:
This does break the `Attributes` tab, should we just enable `createNewAttributesAsGlobal` on the attributes field as well?
Otherwise I would suggestion changing this logic into:
```
inputValue.length > 0 &&
			( createNewAttributesAsGlobal || ! allItems.find(
				( item ) => item.name.toLowerCase() === inputValue.toLowerCase()
			) )
```

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/39827/commits/efd59de7f2a29242c29b71cb9645ca6e7f560058